satpy/etc/readers/gerb_l2_hr_h5.yaml

```diff
@@ -0,0 +1,31 @@
+reader:
+  name: gerb_l2_hr_h5
+  short_name: GERB HR
+  long_name: Meteosat Second Generation Geostationary Earth Radiation Budget L2 High-Resolution
+  description: Reader for the HR product of the Geostationary Earth Radiation Budget instrument
+  status: Beta
+  supports_fsspec: false
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [gerb]
+
+file_types:
+  gerb_l2_hr_h5:
+    file_reader: !!python/name:satpy.readers.gerb_l2_hr_h5.GERB_HR_FileHandler
+    file_patterns: ['{sensor_name}_{seviri_name}_L20_HR_SOL_TH_{sensing_time:%Y%m%d_%H%M%S}_{gerb_version}.hdf']
+
+datasets:
+  Solar_Flux:
+    name: Solar Flux
+    sensor: gerb
+    wavelength: [0.3, 3., 4.]
+    units: W m-2
+    standard_name: toa_outgoing_shortwave_flux
+    file_type: gerb_l2_hr_h5
+
+  Thermal_Flux:
+    name: Thermal Flux
+    sensor: gerb
+    wavelength: [4., 22., 40.]
```

Comment:
I'm surprised these datasets have an associated wavelength. If this information isn't generally used or doesn't make sense for other products you've wanted to add it is optional and doesn't need to be specified.

Reply:
It is indeed not necessary, I'll remove them.